data/kahet/aberrant ships.txt

```diff
@@ -0,0 +1,925 @@
+		"Ka'het Reserve Accumulator"
+
+		"Ka'het Compact Engine" 3
+		"Hyperdrive"
+
+	gun -9 -46.5 "Ka'het Ravager Beam"
+	gun 9 -46.5 "Ka'het Ravager Beam"
+	engine -12.5 27
+	engine 12.5 27
+
+	explode "tiny explosion" 8
+	explode "small explosion" 14
+	explode "medium explosion" 26
+	explode "large explosion" 18
+	"final explode" "nuke explosion"
+	"final explode" "nuke residue fast" 50
+	description "This appears to be some kind of organo-metallic entity, made from a strange mosaic of organic tissue, conventional material, and an exotic matter that defies conventional analysis."
+	description "This variant in particular appears to be growing some sort of tail-like structure, increasing its agility."
+
+
+ship "Aberrant Whiskers" "Aberrant Whiskers (Disable-able)"
+	"display name" `K-IIE "Whiskers"`
+	"uncapturable"
+
+
+ship "Aberrant Trip"
+	"display name" `K-IIF "Trip"`
+	plural "Trips"
+	sprite "ship/aberrant trip"
+	thumbnail "thumbnail/aberrant trip"
+	"never disabled"
+	attributes
+		category "Light Warship"
+		"cost" 5700000
+		"shields" 34600
+		"hull" 33800
+		"required crew" 1
+		"bunks" 1
+		"mass" 444
+		"drag" 6.7
+		"heat dissipation" .65
+		"fuel capacity" 900
+		"ramscoop" 0.2
+		"cargo space" 126
+		"outfit space" 429
+		"weapon capacity" 266
+		"engine capacity" 244
+		"energy capacity" 23000
+		"inscrutable" 1
+		"energy generation" 39
+		"heat generation" 32
+		"shield heat" 6.6
+		"shield generation" 1.8
+		"hull repair rate" 1.4
+		"ka'sei" 1
+		weapon
+			"blast radius" 600
+			"shield damage" 12400
+			"hull damage" 9600
+			"hit force" 3800
+	outfits
+		"Ka'het Ravager Beam" 4
+		"Nucleolysis Beam" 2
+
+		"Ka'het Primary Cooling"
+		"Ka'het Shield Restorer"
+		"Ka'het MHD Generator"
+		"Ka'het Reserve Accumulator"
+		"Anomalous Shield Restorer"
+
+		"Ka'het Compact Engine" 2
+		"Hyperdrive"
+
+	gun -8.5 -42 "Nucleolysis Beam"
+	gun 8.5 -42 "Nucleolysis Beam"
+	gun -25 -31 "Ka'het Ravager Beam"
+	gun 25 -31 "Ka'het Ravager Beam"
+	gun -35.5 -22 "Ka'het Ravager Beam"
+	gun 35.5 -22 "Ka'het Ravager Beam"
+	turret 0 -21
+	turret -13 -3
+	turret 13 -3
+
+	engine -29 38
+	engine 29 38
+	engine -8 57
+	engine 8 57
+
+	explode "tiny explosion" 8
+	explode "small explosion" 14
+	explode "medium explosion" 26
+	explode "large explosion" 18
+	"final explode" "nuke explosion"
+	"final explode" "nuke residue fast" 50
+	description "This appears to be some kind of organo-metallic entity, made from a strange mosaic of organic tissue, conventional material, and an exotic matter that defies conventional analysis."
+	description "Something appears to have gone terribly wrong in the germination of this variant. It is a lopsided, deformed mass of malformed material, which survives only through sheer luck."
+
+ship "Aberrant Trip" "Aberrant Trip (Disable-able)"
+	"display name" `K-IIF "Trip"`
+	"uncapturable"
+
+
+ship "Aberrant Triplet"
+	"display name" `K-IIG "Triplet"`
+	plural "Triplets"
+	sprite "ship/aberrant triplets"
```

Comment:
Pointed improperly- the file's named "triplet", not "triplets".

Reply:
This causes at least the initial defense mission to be unwinnable, because there's an invisible Aberrant flying around shooting you.